protocol/dubbo/dubbo_protocol.go

```diff
@@ -67,9 +69,17 @@ func (dp *DubboProtocol) Export(invoker protocol.Invoker) protocol.Exporter {
 }
 
 func (dp *DubboProtocol) Refer(url common.URL) protocol.Invoker {
+	//default requestTimeout
+	var requestTimeout = config.GetConsumerConfig().RequestTimeout
+
+	requestTimeoutStr := url.GetParam(constant.REQUEST_TIMEOUT, config.GetConsumerConfig().Request_Timeout)
```

Comment:
when is REQUEST_TIMEOUT  set to `url`

Reply:
![image](https://user-images.githubusercontent.com/20589945/71980893-28bbd580-325c-11ea-8e50-23c500151522.png)